wordcounter.rb

```diff
@@ -0,0 +1,30 @@
+class Counter
+  attr_accessor :dictionary, :filepath
```

Comment:
This line defines getter and setter methods, but you don't use them. I don't think you need the setter methods, and `attr_reader` is sufficient. I recommend using the reader methods even internally where you have access to the instance variables.

Ruby has the unfortunate behavior of returning `nil` for unknown instance variables: `@filepaht #=> nil` which can lead to problems that are harder to debug than they should be. Calling an undefined method, on the other hand raises an error immediately.